### DISCUSSION TITLE
AntDesignVue NaiveUI ElementPlus 和原生版本

### DISCUSSION THREAD

**Discussion by z0y1** [vbenjs/vue-vben-admin#4966] [General]
请问下，这4个版本，官方推荐使用哪个版本，是否可以混用？主力开发的是哪个版本？

**Comment by ArthurDarkstone** [comment]
1. 目前三种UI 的样式基本统一，没有推荐的版本，根据实际业务或个人偏好选择即可。
2. 可以混用但不推荐，混用会增加最终构建包的大小，况且一个组件库基本已经能够cover住所有的业务场景了。

**Comment by mynetfan** [comment]
基础框架中所用到的组件与AntDesignVue、NaiveUI、ElementPlus等具体的组件库无关，包括弹窗、抽屉、表单等等也是独立的。所以实际上你无论用哪个版本，基本上也只是在挑选你自己所中意的组件库，本质上区别不大

**Reply by Yang1960** [reply]
怎么使用？怎么全局引入组件库，我用不了，报错

**Comment by frankchen211** [comment]
基础框架使用的是shadcn-vue，如何在app中也使用shadcn-vue，框架似乎没有export shadcn-ui.
